src/hidimstat/clustered_inference.py

```diff
@@ -242,6 +261,13 @@ def clustered_inference(
            Spatially relaxed inference on high-dimensional linear models.
            arXiv preprint arXiv:2106.02590.
     """
+    if memory is not None and not isinstance(memory, str):
+        raise ValueError(
```

Comment:
Is that error not supposed to be handled by `check_memory` line 269?

Reply:
yes.